### PR TITLE
test: add tests for reset command

### PR DESCRIPTION
**Pull request by blaketingyu**
test: add tests for reset command

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -43,7 +44,21 @@
         Recipe recipe = new Recipe(new Name("Aglio Olio"), new CompletionTime(5), new ServingSize(1),
                 ingredients, steps, getTagSet("Western", "Italian"));
 
-        return new Recipe[] {recipe};
+        //Nasi Lemak
+        Ingredient i1 = new Ingredient("fish", 10);
+        Ingredient i2 = new Ingredient("rice", 1, "- s cook the rice");
```
Comment (aidanaden) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java]: This seems incorrect, Ingredient quantifier should not contain a step

```suggestion
        Ingredient i2 = new Ingredient("rice", 1, null);
```

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -43,7 +44,21 @@
...
+
+        Ingredient[] ingredientArr2 = new Ingredient[]{i1, i2};
+        List<Ingredient> ingredients2 = List.of(ingredientArr2);
```
Comment (aidanaden) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:52]: minor: variable name suggestion

```suggestion
        List<Ingredient> nasiLemakIngredients = List.of(ingredientArr2);
```

File: src/test/java/seedu/address/logic/commands/ClearCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for.
+ * {@code ClearCommand}.
+ */
+public class ClearCommandTest {
+    private final Model model = new RecipeBookModelManager(getTypicalRecipeBook(), new UserPrefs());
+    private final Model expectedModel = new RecipeBookModelManager(getTypicalRecipeBook(), new UserPrefs());
+
+    @Test
+    public void execute_showsRequireConfirmationMessage() {
```
Comment (aidanaden) [inline src/test/java/seedu/address/logic/commands/ClearCommandTest.java:23]:
```suggestion
    public void execute_showsRequireConfirmationMessage_success() {
```

File: src/test/java/seedu/address/logic/commands/ResetCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for.
+ * {@code ResetCommand}.
+ */
+public class ResetCommandTest {
+    private final Model model = new RecipeBookModelManager(getTypicalRecipeBook(), new UserPrefs());
+    private final Model expectedModel = new RecipeBookModelManager(getTypicalRecipeBook(), new UserPrefs());
+
+    @Test
+    public void execute_showsRequireConfirmationMessage() {
```
Comment (aidanaden) [inline src/test/java/seedu/address/logic/commands/ResetCommandTest.java:23]:
```suggestion
    public void execute_showsRequireConfirmationMessage_success() {
```

**Review by aidanaden — CHANGES_REQUESTED** [review]
Small nits and name suggestions, also changed the nasi lemak sample recipe's incorrect ingredient values.

Otherwise, LGTM! 🍝

**Review by aidanaden — APPROVED** [review]
Just 1 minor question, overall LGTM! 🍝 🍝 🍝  1 more feature to go!!!

**Review by yusufaine — APPROVED** [review]
Changes looks good to me too! Lessssgeddddiiiitt 🔥

**Pull request merged**